gammapy/maps/tests/test_wcsnd.py

```diff
@@ -660,3 +660,13 @@ def test_reduce():
     m3 = m1.reduce(axis="ax1", keepdims=False)
     assert_allclose(m3.geom.data_shape, (2, 3, 5, 5))
     assert_allclose(m3.data[0][0][0][0], 4.0)
+
+
+def test_to_cube():
+    ax1 = MapAxis.from_nodes([1, 2, 3, 4], name="ax1")
+    ax2 = MapAxis.from_edges([5, 6], name="ax2")
```

Comment:
I guess we require that the new axes have exactly 1 bin? If so maybe add a check in `Map.to_cube()` and document it...